Formula/mpv.rb

```diff
@@ -44,6 +43,13 @@ def install
     # that's good enough for building the manpage.
     ENV["LC_ALL"] = "C"
 
+    # Avoid unreliable macOS SDK version detection
+    # See https://github.com/mpv-player/mpv/pull/8939
+    if OS.mac? && (sdk = MacOS.sdk)
+      ENV["MACOS_SDK"] = sdk.path
```

Comment:
Probably needs some changes to make sure it picks up XCode rather than CLT:
```
Checking for macOS SDK                   : /Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk (version found: 11.0) 
```

Reply:
Looks similar to https://github.com/apple/swift-corelibs-foundation/issues/3198 but that was already fixed in Xcode 13.1